editor.planx.uk/src/@planx/components/Notice/Editor.tsx

```diff
@@ -46,116 +47,96 @@ const NoticeEditor: React.FC<NoticeEditorProps> = (props) => {
+              handleChange({
+                ...values,
                 color,
               });
             }}
-            disabled={props.disabled}
+            disabled={disabled}
           />
           <InputRow>
             <Switch
-              checked={Boolean(props.value.resetButton)}
+              checked={Boolean(values.resetButton)}
               onChange={() =>
-                props.onChange({
-                  ...props.value,
-                  resetButton: !props.value.resetButton,
+                handleChange({
+                  ...values,
+                  resetButton: !values.resetButton,
                 })
               }
               label="Reset to start of service"
-              disabled={props.disabled}
+              disabled={disabled}
             />
           </InputRow>
         </ModalSectionContent>
       </ModalSection>
-      <MoreInformation
-        changeField={(ev: any) => {
-          props.onChange({
-            ...props.value,
-            [ev.target.name]: ev.target.value,
-          });
-        }}
-        definitionImg={props.value.definitionImg}
-        howMeasured={props.value.howMeasured}
-        policyRef={props.value.policyRef}
-        info={props.value.info}
-        disabled={props.disabled}
-      />
+      <MoreInformation formik={formik} disabled={disabled} />
       <InternalNotes
         name="notes"
         onChange={(ev) => {
-          props.onChange({
-            ...props.value,
+          handleChange({
+            ...values,
             notes: ev.target.value,
           });
         }}
-        value={props.value.notes}
-        disabled={props.disabled}
+        value={values.notes}
+        disabled={disabled}
       />
       <ComponentTagSelect
         onChange={(value) =>
-          props.onChange({
-            ...props.value,
+          handleChange({
+            ...values,
             tags: value,
           })
         }
-        value={props.value.tags}
-        disabled={props.disabled}
+        value={values.tags}
+        disabled={disabled}
       />
     </>
   );
 };
 
 const NoticeComponent: React.FC<Props> = (props) => {
-  const formik = useFormik<{ notice: Notice }>({
-    initialValues: {
-      notice: parseNotice(props.node?.data),
-    },
+  const formik = useFormik<Notice>({
+    initialValues: parseNotice(props.node?.data),
     onSubmit: (newValues) => {
       if (props.handleSubmit) {
-        props.handleSubmit({ type: TYPES.Notice, data: newValues.notice });
+        props.handleSubmit({ type: TYPES.Notice, data: newValues });
       }
     },
     validate: () => {},
   });
 
   return (
     <form onSubmit={formik.handleSubmit} id="modal">
-      <NoticeEditor
-        value={formik.values.notice}
-        onChange={(notice) => {
-          formik.setFieldValue("notice", notice);
-        }}
-        disabled={props.disabled}
-      />
+      <NoticeEditor formik={formik} disabled={props.disabled} />
```

Comment:
Again, we're passing the full `formik` instance along here so it's available to `MoreInformation`.